papr/jjb/Jenkinsfile

```diff
@@ -0,0 +1,17 @@
+// XXX: extend openshift/jenkins S2I builder so we can easily bake custom cloud
+// configs in config.xml.tpl so we don't need to `podTemplate` each time
+podTemplate(label: 'papr-slave', cloud: 'openshift', containers: [
```

Comment:
This is some high level magic.

Reply:
It's at least somewhat documented in https://github.com/jenkinsci/kubernetes-plugin. Ideally, this would be hidden from view if we baked some of that config into the image, though I punted on that for now.